main/EDInitMod.F90

```diff
@@ -216,10 +220,12 @@ subroutine zero_site( site_in )
     ! canopy spread
     site_in%spread = 0._r8
 
+    site_in%area_pft(:) = 0._r8
+    site_in%use_this_pft(:) = fates_unset_int
```

Comment:
This is great, I like initializing things to unset.
Also, area_pft looks like it could be unset also.

Reply:
I used 'fates_unset_int' on the use_this_pft as it's an integer, but all the other real numbers above were set to 0._r8 so that seems to be the protocol there. Is there an unset for real numbers?

Reply:
yup, fates_unset_r8, defined after fates_unset_int, se here: https://github.com/NGEET/fates/blob/master/main/FatesConstantsMod.F90#L22